src/main/java/frc/robot/Robot.java

```diff
@@ -20,6 +26,22 @@ public class Robot extends LoggedRobot {
 
   @Override
   public void robotInit() {
+      UsbCamera camera = CameraServer.startAutomaticCapture();
+      camera.setResolution(640, 480);
+
+      CvSink cvSink = CameraServer.getVideo();
+      CvSource outputStream = CameraServer.putVideo("blur", 640, 480);
+
+      Mat source = new Mat();
+      Mat output = new Mat();
+
+      while(!Thread.interrupted()) {
```

Comment:
You need to actually put this in a thread or else it will block the rest of the robot code 😬 

See https://docs.wpilib.org/en/stable/docs/software/vision-processing/roborio/using-the-cameraserver-on-the-roborio.html#advanced-camera-server-program and the variable `m_visionThread` (but please just call it `visionThread`)